addons/test_mail/tests/test_message_composer.py

```diff
@@ -274,14 +274,15 @@ def test_composer_w_template_email_field(self):
         # specific for customer with formatted email
         partner1_email = next(
             mail for mail in self._mails
-            if mail['email_to'] == [tools.formataddr((self.partner_1.name, '"Valid Formatted" <valid.lelitre@agrolait.com>'))]
+            if mail['email_to'] == [tools.formataddr((self.partner_1.name, 'valid.lelitre@agrolait.com'))]
```

Comment:
In commit:

> [IMP] mail: try to better support multi emails when sending mails

It still says here "Mail: sent an email to a broken address" where it is actually fixed I think.

(Side note: messages mention `("Tony" <"Formatted" <tony@e.com>>)` but it seems you are testing with "agrolait" suff? Maybe I missed something).

Reply:
Tests are written to use the same namespace as existing tests (old tests use agrolait). I made comments a bit more generic (and using shorted names / emails just to lessen comment blocks).